packages/wrangler/src/init.ts

```diff
@@ -462,9 +510,14 @@ export async function initHandler(args: ArgumentsCamelCase<InitArgs>) {
 				await mkdir(path.join(creationDirectory, "./src"), {
 					recursive: true,
 				});
-
+				const serviceMetaData = await fetchResult<ServiceMetaDataRes>(
+					`/accounts/${accountId}/workers/services/${fromDashScriptName}`
+				);
+				const defaultEnvironment =
+					serviceMetaData["default_environment"]["environment"];
+				// I want the default environment, assuming it's the most up to date code.
 				const dashScript = await fetchDashboardScript(
-					`/accounts/${accountId}/workers/scripts/${fromDashScriptName}`,
+					`/accounts/${accountId}/workers/services/${fromDashScriptName}/environments/${defaultEnvironment}/content`,
 					{
 						method: "GET",
 					}
```

Comment:
Is the `{method: "GET"}` necessary? Is that not the default for `fetch`?

Reply:
Not particularly.